styles/main.css

```diff
@@ -44,103 +48,152 @@ a {
+}
+
 footer hr {
-    width:50vw;
-    min-width:300px;
-    margin-top:3em;
-    margin-bottom:2em;
-    border-left:0;
-    border-right:0;
+    width: 50vw;
+    min-width: 300px;
+    margin-top: 3em;
+    margin-bottom: 2em;
+    border-left: 0;
+    border-right: 0;
     border-top: 3px solid #8b0000;
     border-bottom: 8px dotted #8b0000;
     border-top-right-radius: 8px;
     border-top-left-radius: 8px;
-    
-}
-*{box-sizing: border-box;
-}
- header div{background-image: url("/images/paper.png");
-            max-width:50px;
-            max-height:50px;}
-
-
-.mugshot {float: right;
-          position: relative;
-          top:-1em;
-          }
-.the {background-color: #F09A9D;}
-header nav {float: right;
-            }
-header nav li{float:left;}
-nav a { text-decoration: none;
-        padding-right: 10px; 
-        padding-top:1em;
-        padding-bottom:0.5em;
-        margin-left:1em;
-        margin-right:1em;
-        display: inline-block;
-       
-               
-}
-header {border-top-style: solid;
-        border-top-color: rgba(0, 0, 0, 0.1);
-        margin-left: -2em;
-        margin-right: -2em;
-}
-header nav {margin-left:2em;
-            margin-right:2em;
-            border-bottom-style: dotted;
-            
-            border-color: rgba(0, 0, 0, 0.3);
-            border-width:1px;
-            margin-top:-4px;
+
 }
-header a:hover {
-  border-top-style:solid;
-  border-width:4px;
-  border-color:yellow;
+
+* {
+    box-sizing: border-box;
+}
+
+.paperclip {
+    background-image: url("../images/paper.png");
+    background-size: contain;
+    width: 50px;
+    height: 50px;
+    position: absolute;
+    top: -10px;
+    right: -7px;
+}
+
+
+.mugshot {
+    float: right;
+    position: relative;
+    top: -1em;
+}
+
+.the {
+    background-color: #F09A9D;
+    clear: right;
+}
+
+header nav ul {
+    float: right;
+}
+
+header nav li {
+    float: left;
+}
+
+nav a {
+    text-decoration: none;
+    padding-right: 10px;
+    padding-top: 1em;
+    padding-bottom: 0.5em;
+    margin-left: 1em;
+    margin-right: 1em;
+    display: block;
+
+    border-top: 4px solid transparent;
+}
+
+header {
+    border-top-width: 4px;
+    border-top-style: solid;
+    border-top-color: rgba(0, 0, 0, 0.1);
+    margin-left: -2em;
+    margin-right: -2em;
 }
-.back-to-top{text-align: right;
-             margin-right:-0.5em;
-             opacity:0.5;
-             position: sticky;
-             bottom: 3.5em;
-             }
 
+header nav {
+    margin-left: 2em;
+    margin-right: 2em;
+    border-bottom-style: dotted;
+
+    border-color: rgba(0, 0, 0, 0.3);
+    border-width: 1px;
+    margin-top: -4px;
+
+    overflow: auto;
+}
+
+header a:hover {
+    border-top-style: solid;
+    border-width: 4px;
+    border-color: yellow;
+}
 
+.back-to-top {
+    text-align: right;
+    margin-right: -0.5em;
+    opacity: 0.5;
+}
+.back-to-top a {
+    position: fixed;
+    bottom: 3.5em;
+}

```

Comment:
This is the correct place for these two declarations.  Note: `position: fixed` instead of `position: sticky`